docs/blog/_posts/2021-04-21-scala3-rc3.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-page
+title: Scala 3.0.0-RC3 – bug fixes for 3.0.0 stable
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2021-04-21
+---
+
+Hello! With this blog article, we would like to announce the release of Scala 3.0.0-RC3. Some critical bugs were discovered during the previous release cycle the fixes to which had to be included in 3.0.0 stable. Therefore, we are having RC3 to give the community time to test our fixes.
+
+This release also impacts the release date for 3.0.0 stable. 3.0.0 stable will get a green light if no critical bugs are discovered until May 12th. Otherwise we may need to have an RC4.
+
+<!--more-->
+
+# Bug fixes included
+- Backport: Fix type test for trait parameter arguments [#12066](https://github.com/lampepfl/dotty/pull/12066)
```

Comment:
I suggest removing all those "Backport:" prefixes. From the point of view of the branches in the dotty repo, they make sense. But for users, it doesn't, since there's no further release.